src/patternfly/components/LogViewer/log-viewer.scss

```diff
@@ -85,6 +96,24 @@
     word-break: break-all;
   }
 
+  &.pf-m-nowrap {
+    --pf-c-log-viewer__text--WhiteSpace: var(--pf-c-log-viewer--m-nowrap__text--WhiteSpace);
+
+    .pf-c-log-viewer__list {
```

Comment:
Also, the same as `.pf-c-log-viewer__index`, could you also please apply these styles directly to the `.pf-c-log-viewer__list`? The style when wrapping lines is incorrect currently, but after taking this class and `.pf-c-log-viewer__index` out of the `.pf-m-nowrap`, the problem could be solved.

<img width="829" alt="Screen Shot 2021-11-08 at 1 18 19 PM" src="https://user-images.githubusercontent.com/37624318/140796790-fd7296de-06e1-467b-9cc9-2f667825acd2.png">